packages/vsce/src/commands/setCICSRegionCommand.ts

```diff
@@ -19,147 +19,205 @@ import { FilterDescriptor } from "../utils/filterUtils";
 import * as regionUtils from "../utils/lastUsedRegionUtils";
 import { InfoLoaded, ProfileManagement } from "../utils/profileManagement";
 
+/* Localized templates / labels */
+const LAST_USED_TEMPLATE = (region: string, plex: string, profile: string) =>
```

Comment:
Discussion point :-)

I'm not sure I like these all being moved up here.
The reason is:
1. I think it makes the code harder to read because you have a big clump of stuff here and then as you read further down you're wondering about what's in the constant
2. The VS Code localisation is very lightweight and only adds 8 characters to each string `l10n.t(`...`)` so it's not much of a problem to keep it in line.

I think with Java code we would often extract string contants like this - I'm not convinced it's what people do in TypeScript.